charts/csi-vxflexos/templates/node.yaml

```diff
@@ -230,17 +230,19 @@ spec:
           volumeMounts:
             - name: driver-path
               mountPath: {{ .Values.kubeletConfigDir }}/plugins/vxflexos.emc.dell.com
-            - name: disks-path
-              mountPath: {{ .Values.kubeletConfigDir }}/plugins/vxflexos.emc.dell.com/disks
               mountPropagation: "Bidirectional"
             - name: volumedevices-path
               mountPath: {{ .Values.kubeletConfigDir }}/plugins/kubernetes.io/csi/volumeDevices
               mountPropagation: "Bidirectional"
             - name: pods-path
               mountPath: {{ .Values.kubeletConfigDir }}/pods
               mountPropagation: "Bidirectional"
-            - name: noderoot
-              mountPath: /noderoot
+            - name: scaleio-path-bin
```

Comment:
Do these changes require changes in the driver?

Reply:
Not in the driver, but there are already merged corresponding changes in goscaleio: https://github.com/dell/goscaleio/pull/190